.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.md
```

Comment:
Good work adding node_modules to ignored files

- Kindly modify the name in your `.test.md` to match the one that we want to ignore `test.md`. So in this way the file will be correctly ignored on GitHub 💯